src/__experimental__/components/search/search.spec.js

```diff
@@ -30,27 +30,44 @@ describe('Search', () => {
       }, renderWrapper({ value: '' }, mount));
     });
 
+    it('applies the default width when the user does not specify a width', () => {
```

Comment:
Neither of these tests asserts anything about the width? Something like below might work

```
assertStyleMatch({
        borderBottom: '2px solid #CCD6DA',
        display: 'inline-flex',
        fontSize: '14px',
        fontWeight: '700',
       width: '100%'
      }, renderWrapper({ value: '' }, mount));
```

Reply:
Done.